web-console/knikubevirt/Affinity/affinity.md

```diff
@@ -0,0 +1,61 @@
+# Affinity
+
+### About Affinity
+Affinity allows the user to create a relationship between two resources when scheduling them on the cluster.
+There are two high level relationship types:
+Affinity - attaching a group of resources together, making them run together in the same environment.
+Anti-affinity - creating a rejection relationship between a group of resources, thus avoiding scheduling any of them in the same environment.
+
+Besides defining if we want to attach or reject resources to one another, we can choose to "force" this rule we are creating.
+"Forcing" a rule means that the conditions of this rule must met, even if it means that some resources just won't be scheduled at all.
+
+### a few techincal words
+Affinity rules are Pod Properties, written in the pod deployment file. When a pod is scheduled, the deployment file states that there is an affinity rule condition which needs to be met.
+Applying an affinity rule to a resources is done via labels. Objects in OpenShift are ephemeral, and the way to target the same object which keeps crashing and regenerating is by its label.
+
+
+## Affinity rules - list page
+
+![Affinity list](img/D2-0-0.jpg)
+All affinity rules within the system can be viewed and managed from this page. It is located in the main navigation, under 'Workloads' 
+
+## Creating an Affinity rule from Affinity Rules page
+
+Clicking on the "Add Affinity rule" button at the top of the affinity list will open up the "Add Affinty rule" modal.
+
+![Affinity rule modal - default](img/D2-1-0.jpg)
+Going top to bottom, the user will choose to create an affinity (attachment), which is pre-selected, or change to Anti-affinity (rejection).
+Then, the user will select the resources he wishes to apply that rule to by adding their representative labels to the "Select resources by label" field.
+
+
+![Force rule](img/D2-1-1.jpg)
```

Comment:
Hey Ran, for these modals I think we'll want to switch them to ones without the header. 
https://www.patternfly.org/v4/documentation/react/components/modal/
It will be nice to keep that secondary header helper text though.